ansible/roles/cyhy_feeds/vars/main.yml

```diff
@@ -1,2 +1,36 @@
 ---
-# vars file for cyhy_feeds
\ No newline at end of file
+# vars file for cyhy_feeds
+
+# cyhy-feeds config
+config: "{{ lookup('aws_ssm', '/cyhy/feeds/config') }}"
+
+# cyhy-feeds public GPG key
+public_gpg_key: "{{ lookup('aws_ssm', '/cyhy/feeds/gpg/public') }}"
+# cyhy-feeds private GPG key
+private_gpg_key: "{{ lookup('aws_ssm', '/cyhy/feeds/gpg/private') }}"
+# NSD public GPG key
+nsd_public_gpg_key: "{{ lookup('aws_ssm', '/cyhy/feeds/gpg/nsd_public') }}"
+
+# cyhy-feeds GPG trust
+trust: "{{ lookup('aws_ssm', '/cyhy/feeds/gpg/trust') }}"
```

Comment:
Same here, I'd prefer to see this called `gpg_trust` to improve readability

Reply:
Changed in 5a7097c.